<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produkt_info</class>
 <widget class="QDialog" name="produkt_info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#produkt_info{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(61, 196, 47, 255), stop:1 rgba(23, 88, 0, 255));}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="8" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(255, 187, 0, 255), stop:1 rgba(182, 97, 46, 255));color:white;</string>
     </property>
     <property name="text">
      <string>Zamknij</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLabel" name="label_ilosc">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0 szt.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="label_netto">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00zł</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QLabel" name="label_brutto">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00.00zł</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_brutto_opis">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cena Brutto:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_ilosc_opis">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STAN MAGAZYNOWY:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_netto_opis">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cena Netto:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_opis">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OPIS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="label_nazwa">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NAZWA PRODUKTU</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_info">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>STAN POŁĄCZENIA</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_vat_opis">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>STAWKA VAT:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_vat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
